bin/templates/cordova/lib/build.js

```diff
@@ -372,6 +409,7 @@ function parseOpts(options, resolvedTarget) {
             if (multiValueArgs[flagName] && !flagValue) {
                 flagValue = options[i + 1];
                 ++i;
+                console.warn('A value is expected for the flag: ' + flagName);
```

Comment:
I think this warning will show when not using an = for the flag value (generally for flags with values, you're allowed to do: --foo=bar *or* --foo bar